src/mocks/k8s.io/client-go/kubernetes/typed/core/v1/pod_interface.go

```diff
@@ -0,0 +1,855 @@
+// Code generated by mockery v2.15.0. DO NOT EDIT.
```

Comment:
is this code supposed to be checked in?

Reply:
For now, yes, of course, it is needed by the tests.